### DISCUSSION TITLE
Fetch API

### DISCUSSION THREAD

**Discussion by To1iK** [sebastienros/jint#1253] [General]
Hello
Is it possible to use Fetch API from jint?

It is like [here](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)

**Comment by lahma** [comment]
Some discussion here: https://github.com/sebastienros/jint/issues/442 . Basically you need to inject the APIs you need by registering TypeReference and implementing them.

**Reply by To1iK** [reply]
So, there is no fetch function in jint, but there is something better..  I can make it by c#. Thanks. It's cool. And what about Arrow functions?
It is like [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
Is it posible to implement them too?

**Reply by lahma** [reply]
Arrow functions and many more new functionality can be found in v3 betas, see readme.
